api-gateway/src/test/java/com/petclinic/bffapigateway/presentationlayer/ApiGatewayControllerTest.java

```diff
@@ -1846,6 +1853,39 @@ void shouldGetBadRequestWhenInvalidPetIdToRetrieveScheduledVisits() {
                 .jsonPath("$.message").isEqualTo(expectedErrorMessage);
     }
 
+    @Test
+    void deleteVisitById_visitId_shouldSucceed(){
```

Comment:
[Nitpick]: should be named deleteVisitByVisitId...

Reply:
I do agree in terms of keeping consistency in the code base. However I did learn this naming convention in class